vars/buildManifest.groovy

```diff
@@ -1,9 +1,10 @@
 void call(Map args = [:]) {
     sh (([
-        args.script ?: './build.sh',
+        args.dryRun ? 'echo ./build.sh' : './build.sh',
```

Comment:
Did we need to modify the build.sh to add `dryRun` arg or somewhere else?

Reply:
In this case "dry run" = "replace command with echo command ..." but yes, we could extend `build.sh`, `assemble.sh` and `test.sh` to take `--dry-run`. I'm not sure it's much better though.